src/OBD9141.h

```diff
@@ -57,7 +57,7 @@
 // The ECU might not push all bytes on the bus immediately, but wait several ms
 // between the bytes, this is the time allowed per byte for the answer
 
-#define OBD9141_WAIT_FOR_REQUEST_ANSWER_TIMEOUT (30 + 10)
+#define OBD9141_WAIT_FOR_REQUEST_ANSWER_TIMEOUT (30 + 20)
```

Comment:
Slight increase in this timeout, KWP may require some different timing. In normal operation the code never reaches this timeout so it doesn't negatively affect any successful operation because we return after reading all the expected bytes.